# Solution:
# 
#   excep(0):
#     try:
#       try:
#         print(7/0) -> error
#       will catch ZeroDivisionError and print "zero"
#       finally will raise an Exception with the message "False"
#     will catch Exception as m and print "False"

#  zero
#  False
#  type
#  True
#  None